PowerUp/app/src/main/res/values/strings.xml

```diff
@@ -109,7 +109,7 @@
     <string name="memory_label">Does the tile on the left match the tile that appeared before it?</string>
     <string name="no">NO</string>
     <string name="yes">YES</string>
-    <string name="tutorial_1">Shoot the viruses with the syringe!</string>
+    <string name="tutorial_1">Shoot the viruses with the syringe by tapping it when the virus appears infront of it!</string>
```

Comment:
I don't really feel this instruction is clearer than the previous one.
@geekanamika What do you think.